xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -763,7 +763,7 @@ std::string CGUIWindowVideoBase::GetResumeString(const CFileItem &item)
 
 bool CGUIWindowVideoBase::ShowResumeMenu(CFileItem &item)
 {
-  if (!item.m_bIsFolder && !item.IsPVR())
+  if (!item.m_bIsFolder && !item.IsPVR() || item.IsPlugin())
```

Comment:
The root cause of this was the new movie item not being created from the original CFileItemPtr but it used the video info tag to populate it. This used some weird logic to work out if it was a folder or not. If there was no FileAndPath set in the video info tag then it was assumed it was a dir and set the path to / and marked it as a Folder.

I made a small change to populate the movie item from the base CFileItemPtr, tjis uses the original items IsFolder value.

This is now working.